docs/source/download.rst

```diff
@@ -110,6 +110,10 @@ examples on the :doc:`documentation page <documentation>`.
 Git instructions for developers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Cython >= 0.17 is required to build Firedrake. Install it using pip ::
+
+ pip install "Cython>=0.17"
```

Comment:
Is Cython 0.17 still sufficient? I seem to recall that 0.19 or so was required for some of the features we use.